File: XKS.Desktop/Controller/SessionController.cs
using XKS.View;

namespace XKS.Controller
{
	public sealed class SessionController
	{
		private readonly SessionView _view;

		public SessionController(SessionView view)
		{
			_view = view;
		}

		public void SetDefaultFocus()
		{
			_view.QuestionEntryBox.GrabFocus();
		}
	}
}

File: cpp/xks/controller/SessionController.cpp
/**
 * Copyright 2020 Vitalijus Dobrovolskis
 *
 * Created by Vitalijus Dobrovolskis on 06.02.2020
 */

#include "SessionController.h"

xks::controller::SessionController::SessionController(sptr<SessionView> view)
		: view(std::move(view)) {}
